src/families/bitcoin/js-signOperation.ts

```diff
@@ -73,14 +73,10 @@ const signOperation = ({
 
         const segwit = isSegwitDerivationMode(account.derivationMode);
 
-        // FIXME Call to explorer needed to set timestamp (https://ledgerhq.atlassian.net/browse/LL-7539)
-        // cf. https://github.com/LedgerHQ/lib-ledger-core/blob/fc9d762b83fc2b269d072b662065747a64ab2816/core/src/wallet/bitcoin/transaction_builders/BitcoinLikeUtxoPicker.cpp#L150-L154
-        /*
-        const hasTimestamp = networkParams.usesTimestampedTransaction;
+        const hasTimestamp = currency.id === "peercoin";
```

Comment:
Why removing this? Could've stayed generic I think, but no big deal.

Reply:
I kinda agree with @haammar-ledger here

Reply:
I agree with @haammar-ledger , ideally (and it's not yet the case), each coin should have its folder that defines all its "specifics" and only this folder should have such "if",
that said it's not the only place where we have these I feel.

Reply:
The btc-js impl already has such mechanisms actually:
- https://github.com/LedgerHQ/ledger-live-common/blob/16bc0befa95687030b0ced54c3db30369836390a/src/families/bitcoin/logic.ts#L149-L194
- https://github.com/LedgerHQ/ledger-live-common/blob/master/src/families/bitcoin/networks.ts

They're legacy, could need cleanup for some altcoins.